modules_artemis/__HELPERS/mobs.dm

```diff
@@ -0,0 +1,409 @@
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/spines, spines_list)
+	if(!legs_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/legs, legs_list)
+	if(!body_markings_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/body_markings, body_markings_list)
+	if(!wings_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/wings, wings_list)
+	if(!tentacles_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/tentacles, tentacles_list)
+
+	//For now we will always return none for tail_human and ears.
+	return(list("mcolor" = pick("FFFFFF","7F7F7F", "7FFF7F", "7F7FFF", "FF7F7F", "7FFFFF", "FF7FFF", "FFFF7F"), "tail_lizard" = pick(tails_list_lizard), "tail_human" = "None", "wings" = "None", "snout" = pick(snouts_list), "horns" = pick(horns_list), "ears" = "None", "frills" = pick(frills_list), "spines" = pick(spines_list), "body_markings" = pick(body_markings_list), "legs" = "Normal Legs"))
+
+/proc/random_hair_style(gender)
+	switch(gender)
+		if(MALE)
+			return pick(hair_styles_male_list)
+		if(FEMALE)
+			return pick(hair_styles_female_list)
+		else
+			return pick(hair_styles_list)
+
+/proc/random_facial_hair_style(gender)
+	switch(gender)
+		if(MALE)
+			return pick(facial_hair_styles_male_list)
+		if(FEMALE)
+			return pick(facial_hair_styles_female_list)
+		else
+			return pick(facial_hair_styles_list)
+
+/proc/random_unique_name(gender, attempts_to_find_unique_name=10)
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		if(gender==FEMALE)
+			. = capitalize(pick(first_names_female)) + " " + capitalize(pick(last_names))
+		else
+			. = capitalize(pick(first_names_male)) + " " + capitalize(pick(last_names))
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_unique_lizard_name(gender, attempts_to_find_unique_name=10)
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		. = capitalize(lizard_name(gender))
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_unique_plasmaman_name(attempts_to_find_unique_name=10)
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		. = capitalize(plasmaman_name())
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_skin_tone()
+	return pick(skin_tones)
+
+var/list/skin_tones = list(
+	"albino",
+	"caucasian1",
+	"caucasian2",
+	"caucasian3",
+	"latino",
+	"mediterranean",
+	"asian1",
+	"asian2",
+	"arab",
+	"indian",
+	"african1",
+	"african2"
+	)
+
+var/global/list/species_list[0]
+var/global/list/roundstart_species[0]
+
+/proc/age2agedescription(age)
+	switch(age)
+		if(0 to 1)
+			return "infant"
+		if(1 to 3)
+			return "toddler"
+		if(3 to 13)
+			return "child"
+		if(13 to 19)
+			return "teenager"
+		if(19 to 30)
+			return "young adult"
+		if(30 to 45)
+			return "adult"
+		if(45 to 60)
+			return "middle-aged"
+		if(60 to 70)
+			return "aging"
+		if(70 to INFINITY)
+			return "elderly"
+		else
+			return "unknown"
+
+/*
```

Comment:
why is this code grouped together with character randomization procs?
a new file dedicated to admin tool helpers would be a better choice IMO